### PR TITLE
Popup Animation In

### PR DESCRIPTION
**Pull request by cfpwastaken**
Adds an animation for popups to animate in
Prob adding more animations idk

### CODE REVIEW

**Comment by cfpwastaken** [comment]
`git checkout f-animation` please so you can look at the animation itself.

**Review by j0code — COMMENTED** [review]
pls no

**Comment by cfpwastaken** [comment]
What??

File: styles/style.css [hunk 1]
```diff
@@ -368,6 +368,19 @@ code {
...
 	transform: translate(-50%, -50%);
 	border: 1px solid var(--background-tertiary);
+	animation-name: popIn;
+	animation-duration: 2s;
+}
+
+@keyframes popIn {
+	from {
+		max-height: 1px;
+		max-width: 1px;
```
Comment (j0code) [inline styles/style.css:378]:
```suggestion
		filter: brightness(0);
		transform: translate(-50%,-50%) scale(0);
```

File: styles/style.css [hunk 2]
```diff
@@ -368,6 +368,19 @@ code {
...
+	}
+	to {
+		max-height: 9999px;
+		max-width: 9999px;
```
Comment (j0code) [inline styles/style.css:382]:
```suggestion
		filter: brightness(1);
		transform: translate(-50%,-50%) scale(1);
```

**Review by j0code — CHANGES_REQUESTED** [review]
fix

**Comment by cfpwastaken** [comment]
idk gonna just commit it eh

**Comment by cfpwastaken** [comment]
@j0code please approve it so this thing doesn't annoy me

**Pull request merged**